### ISSUE TITLE
Why use softmax instead of sigmoid?

### ISSUE THREAD

**Issue by evbtst** [seoungwugoh/RGMP#17]
line 66: `msv_E2[sc] = upsample(F.softmax(e2[0], dim=1)[:,1].data.cpu(), (h,w))`

line 71: `msv_E2[sc] = F.softmax(e2[0], dim=1)[:,1].data.cpu()`

If you run these lines for each object (Propagate_MS is inside the loop `for o in range(num_objects)` ) , why use softmax instead of sigmoid?

**Comment by seoungwugoh** [comment]
Hi @evbtst,
We perform 2-way (FG & BG) softmax for the binary segmentation. The similar operation can be implemented using the sigmoid. But, we simply choose to use the softmax.
